app/components/Header/index.js

```diff
@@ -28,6 +28,36 @@ function Header() {
   );
 }
 
+function Header() {
+  return(
+    <div className="pos-f-t">
+    <div className="collapse" id="navbarToggleExternalContent">
+      <div className="bg-inverse text-white">
+        <Link className="text-muted" to ="/">Home</Link>
```

Comment:
Give these className=`nav-item` and remove the `<br>` -- bootstrap does the inlining work for us